ngraph_bridge/ngraph_executor.cc

```diff
@@ -240,190 +219,169 @@ Status NGraphExecutor::GetNgExecutable(
     return errors::Internal("Backend not available: ", m_op_backend_name);
   }
 
-  lock_guard<mutex> lock(m_mutex);
-
-  std::stringstream signature_ss;
-  std::vector<TensorShape> input_shapes;
-  std::vector<const Tensor*> static_input_map;
-  TF_RETURN_IF_ERROR(ComputeSignature(tf_input_tensors, input_shapes,
-                                      static_input_map, signature_ss));
-  string signature;
-  signature = signature_ss.str();
-
-  NGRAPH_VLOG(5) << "Computed signature: " << signature;
-
-  cache_hit = false;
-  auto it = m_ng_exec_map.find(signature);
-
-  // Translate the TensorFlow graph to nGraph.
-  if (it == m_ng_exec_map.end()) {
-    // Measure the current total memory usage
-    long vm, rss, vm0, rss0;
-    MemoryProfile(vm0, rss0);
-
-    NGRAPH_VLOG(1) << "Compilation cache miss: " << m_node_name;
-
-    std::shared_ptr<ngraph::runtime::Executable> evicted_ng_exec;
-    string serialized_ng_func;
-    std::shared_ptr<ngraph::Function> ng_function;
-    if (!m_do_aot) {
-      TF_RETURN_IF_ERROR(Builder::TranslateGraph(input_shapes, static_input_map,
-                                                 m_graph.get(), ng_function));
-      ng_function->set_friendly_name(m_node_name);
-      int json_indentation = 4;
-      serialized_ng_func = ngraph::serialize(ng_function, json_indentation);
-    } else {
-      auto itr = m_aot_functions.find(signature);
-      if (itr == m_aot_functions.end()) {
-        return errors::Internal(
-            "Expected to find AOT precompiled ng function of signature: ",
-            signature);
-      }
-      serialized_ng_func = itr->second;
-    }
+  cache_hit = true;
```

Comment:
We don't have to set `cache_hit = true;`. The callback should be setting it to whatever needs to be set

Reply:
If item is found in the cache then callback will never be called, hence setting true by default